dvc/repo/checkout.py

```diff
@@ -50,7 +50,7 @@ def _checkout(
         total=total, unit="file", desc="Checkout", disable=total == 0
     ) as pbar:
         for stage in stages:
-            if stage.locked:
+            if stage.locked and warn:
```

Comment:
We can actually delete this warning, as it is a leftover from some old logic that is no longer in place.

Reply:
I see, thanks. I'll get it deleted in my next commit.